releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -1,5 +1,10 @@
 ## Azure App Configuration Extension release notes
 
+> **DEPRECATION NOTICE**  
+> This task is now available as a built-in task for Azure Pipelines under the new name, **Azure App Configuration Export**. This means you no longer need to install it from Marketplace. While you can continue using the Marketplace version, please note that it will no longer receive updates. Transitioning to the new built-in task requires minimal effort, as it supports all the features and functionalities of the Marketplace version. In most cases, it is as simple as updating the task name and version in your pipeline.
```

Comment:
```suggestion
> This pipeline task is now available as a built-in task for Azure Pipelines under the new name, **Azure App Configuration Export**. This means you no longer need to install it from Marketplace. While you can continue using the Marketplace version, please note that it will no longer receive updates. Transitioning to the new built-in task requires minimal effort, as it supports all the features and functionalities of the Marketplace version. In most cases, it is as simple as updating the task name and version in your pipeline.
```